### PR TITLE
fix(ftux): ux and bug fixes

### PR DESCRIPTION
**Pull request by neurosnap**
Mostly UX fixes.

- sc-12926

### CODE REVIEW

File: src/ui/pages/create-project-git.tsx [hunk 1]
```diff
@@ -1240,7 +1258,7 @@ export const CreateProjectGitSettingsPage = () => {
             htmlFor="envs"
             feedbackVariant={envErrors.length > 0 ? "danger" : "info"}
             feedbackMessage={envErrors.map((e) => e.message).join(". ")}
-            description="Environment Variables (each line is a separate variable in format: ENV_VAR=VALUE)."
+            description="Add any additional required variables, such as API keys, KNOWN_HOSTS setting, etc. Each line is a separate variable in format: ENV_VAR=VALUE."
```
Comment (madhuravius) [inline src/ui/pages/create-project-git.tsx:1261]: if descriptions support `pre` type elements, we should use that for the caps here

Comment (madhuravius) [reply src/ui/pages/create-project-git.tsx:1261]: like `<span> .... <pre>SOMETHING=true</pre></span>` instead of plaintext

Comment (neurosnap) [reply src/ui/pages/create-project-git.tsx:1261]: @joeclayallday asked me to remove those at one point.

**Review by madhuravius — APPROVED** [review]
nitpick on better descriptions

lgtm otherwise, mostly copy edits it seems

**Pull request merged**